src/test/java/testng/LongSumCalTest.java

```diff
@@ -7,10 +7,9 @@
 public class LongSumCalTest extends BaseTest{
 
     @Test(dataProvider = "valuesForSumTest")
-    public void plusTest(long a, long b, long expectedValue) throws InterruptedException{
+    public void plusTest(long a, long b, long expectedValue) {
         long result = calculator.sum(a, b);
```

Comment:
actualResult ?

Reply:
Thank you, corrected.
Changes will be seen in the following commit.